### PR TITLE
Makes the AI playable.

### PR DESCRIPTION
**Pull request by DominikPanic**
:cl: LaKiller8
add: The AI is now playable, also fixed the last interaction issues with certain machines.
/:cl:

Fixes: #2183

### CODE REVIEW

**Comment by Rohesie** [comment]
We'll need AI guidelines and rules before we can merge this, but it can be tested.

File: code/modules/mob/living/silicon/ai/ai_verbs.dm [hunk 1]
```diff
@@ -237,4 +237,63 @@
...
+		return
+
+	if(input(src, "Are you sure you want to announce your laws[radiomod ? " over the [radiomod] channel" : ""]?", "State Laws", "Yes", "No") != "Yes")
+		return
+
+	say("[radiomod] Current Active Lawset:")
+
+	var/delay = 1 SECONDS
+	for(var/i in laws)
+		addtimer(CALLBACK(src, .say, "[radiomod] [i]"), delay)
```
Comment (spookydonut) [inline code/modules/mob/living/silicon/ai/ai_verbs.dm]:
```suggestion
		addtimer(CALLBACK(src, .proc/say, "[radiomod] [i]"), delay)
```

Comment (DominikPanic) [reply code/modules/mob/living/silicon/ai/ai_verbs.dm]: code\modules\mob\living\silicon\ai\ai_verbs.dm:269:error: .proc/say: undefined type path
it either won't compile, or won't lint properly.

Comment (spookydonut) [reply code/modules/mob/living/silicon/ai/ai_verbs.dm]: so use whatever the basepath for say() is, /mob/living/.proc/say or whatever it is

Comment (spookydonut) [reply code/modules/mob/living/silicon/ai/ai_verbs.dm]: lint is complaining because its not typesafe and this could potentially runtime

**Comment by Rohesie** [comment]
Two requests:
1) Give it a long cooldown for AI announcements, as it can get really spammy. Or remove the "attention, new command report created" sound. That made sense when it was an admin mob, but not now.
2) In /tg/ AIs can bolt down unpowered doors. Not sure if they can here, but if they can, they shouldn't.

**Comment by DominikPanic** [comment]
```
In /tg/ AIs can bolt down unpowered doors. Not sure if they can here, but if they can, they shouldn't.
```
that was already fixed

File: code/modules/mob/living/silicon/ai/ai_verbs.dm [hunk 2]
```diff
@@ -237,4 +241,63 @@
...
+		return
+
+	if(input(src, "Are you sure you want to announce your laws[radiomod ? " over the [radiomod] channel" : ""]?", "State Laws", "Yes", "No") != "Yes")
+		return
+
+	say("[radiomod] Current Active Lawset:")
+
+	var/delay = 1 SECONDS
+	for(var/i in laws)
+		addtimer(CALLBACK(src, /atom/movable.proc/say, "[radiomod] [i]"), delay)
```
Comment (spookydonut) [inline code/modules/mob/living/silicon/ai/ai_verbs.dm]:
```suggestion
		addtimer(CALLBACK(src, /atom/movable/.proc/say, "[radiomod] [i]"), delay)
```

**Pull request merged**